src/Plutus/Certification/Synchronizer.hs

```diff
@@ -176,7 +176,7 @@ monitorWalletTransactions eb args = withEvent eb MonitorTransactions $ \ev -> do
       return []
     handleResponse (Right transactions) = return transactions
 
-type CertificationProcess m = DB.ProfileId -> UUID -> m DB.Certification
+type CertificationProcess m = DB.ProfileId -> UUID -> m DB.L1CertificationDTO
```

Comment:
L1 certificates created by an auditor subscriber are not broadcasted by IOG and should just be sent to the user for them to broadcast